[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\fakoy\\AppData\\Local\\Temp\\quarto-session279e9d467c3265ab'\n"]
